### PR TITLE
issue#2

### PR DESCRIPTION
**Pull request by james-aguirre**
Kapture 2023-04-26 at 14 06 10
Kapture 2023-04-26 at 14 09 38

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -89,3 +94,39 @@ li {
...
+
+.by-popularity {
+  background-color: #562b81;
+  height: 4.2rem;
+  width: 11.5rem;
+  border-radius: 0.5rem;
+  margin-top: 3rem;
+  font-size: 1.3rem;
+  padding-bottom: 0.7rem;
+}
```
Comment (shawnkost) [inline css/styles.css]: You repeat a lot of CSS between these 3 different selectors. I recommend creating a new class with all the css that gets applied to each of these, and giving that class to these 3 elements.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done James! One change requested here.

Also looking at the gif, the plus icon looks a little weird when it's laid out like that. I recommend having it align vertically with the text, so it looks more clean, rather than having it like up to the corner. I won't require you to fix this for the issue to be marked "complete", but just giving my opinion.

**Comment by james-aguirre** [comment]
Kapture 2023-04-26 at 14 31 48

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**